### ISSUE TITLE
Type documentWithCaptionMessage

### ISSUE THREAD

**Issue by borsatti2** [allburov/whatsapp-http-api#345]
I discovered a bug that I needed to fix urgently.
When someone sends a document with caption, its type is "documentWithCaptionMessage" and it doesn't come with a URL or anything.
So I can't access this file.
Can you check this for us?
![image](https://github.com/devlikeapro/whatsapp-http-api/assets/6342776/930114c6-d309-4ca7-9f0a-d93037691e30)

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by allburov** [comment]
Hi! Fixed in `2024.5.10` version!

That's the power :muscle: of the label :point_down:, I even didn't have a coffee before starting fixing it :laughing: :coffee: 

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by allburov** [comment]
Btw, also added the caption for all messages with captions to `body` field, so it works the same way as for WEBJS engine
```json
{
  "event": "message",
  "session": "default",
  "payload": {
     ...
    "body": "Check this out!", <<<<<<
    "from": "11111111111@c.us",
    "hasMedia": true,
    "media": {
      "url": "http://localhost:3000/api/files/true_11111111111@c.us_AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA.jpg",
      "mimetype": "image/jpeg",
      "filename": null
    },
    ...
  }
}
```

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
